disnake/types/onboarding.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: MIT
+
+from typing import List, Literal, TypedDict
+
+from .emoji import Emoji
+from .snowflake import Snowflake, SnowflakeList
+
+OnboardingPromptType = Literal[0, 1]
+
+
+class OnboardingPromptOption(TypedDict):
+    id: Snowflake
+    title: str
+    description: str
```

Comment:
`description` is nullable, according to the docs - I haven't seen that happen, but it should be handled regardless

Reply:
07d839b85a81f1facceaf2323833b3ab7af85ec7